ui/scripts/modal.js

```diff
@@ -0,0 +1,68 @@
+export default class Modal {
+    // @todo: will need to add trailer url later
+    constructor(title) {
+        this.title = title;
+        // this.url = url;
+    }
+
+    fillInModal() {
+        console.log("hej!")
+        const title = document.querySelector(".modal-title");
+        title.textContent = this.title;
+
+        // @todo: will have a dynamic link to the trailer later
+        const body = document.querySelector(".modal-body");
+        const trailer = this.createTrailer("https://www.youtube.com/embed/LtNYaH61dXY?si=j-aoQsvgBPQFIfBg");
```

Comment:
I assume we'll need to handle the case where trailer is not available; but we can do it later.